DOCS-99: Rewrite SMB "Configuring Authenticated Access with Local Use… (#1633)
freenas/freenas-docs: userguide/sharing.rst

home and small networks as it does not scale well if many user accounts
are needed.

To configure authenticated access for an SMB share, first create a
:ref:`group <Groups>` for all the SMB user accounts in %brand%. Go to
:menuselection:`Accounts --> Groups`
and click |ui-add|. Use a descriptive name for the group like
:samp:`local_smb_users`.

> Next, configure the SMB share dataset to assign permissions to this new

Reviewer: Avoid "Next" and step introducers.
```suggestion
Configure the SMB share dataset with permissions for this new
```

group. When :ref:`creating a new dataset <Adding Datasets>`, set the
:guilabel:`Share Type` to *SMB*. After the dataset is created, open the
dataset :ref:`Access Control List (ACL) <ACL Management>` and add a new

> entry. Set :guilabel:`Who` to *Group* and select the SMB user accounts

Reviewer: ```suggestion
entry. Set :guilabel:`Who` to *Group* and select the SMB
```

group for the :guilabel:`Group`. Finish

> :ref:`defining the permissions <ACE Permissions>` for the SMB user

Reviewer: As above, "SMB user accounts group" is confusing.

accounts group. Any :ref:`members of this group <Groups>` now have

> authenticated access to the dataset.

Reviewer: s/authenticated access/access/
